_msprimemodule.c

```diff
@@ -2193,6 +2193,42 @@ RecombinationMap_init(RecombinationMap *self, PyObject *args, PyObject *kwds)
     return ret;
 }
 
+static PyObject *
+RecombinationMap_position_to_mass(RecombinationMap *self, PyObject *py_position)
+{
+    PyObject *ret = NULL;
+    double position;
+
+    if (RecombinationMap_check_recomb_map(self) != 0) {
+        goto out;
+    }
+    if (!PyArg_ParseTuple(py_position, "|d", &position)) {
```

Comment:
The "|d" makes position an optional argument, which we don't want here. Just get rid of the |